R/effectivemass.R

```diff
@@ -174,7 +176,7 @@ fit.effectivemass <- function(cf, t1, t2, useCov=FALSE, replace.na=TRUE, boot.fi
   }
 
   # cf here is a bootstrapped effective mass and $t is the matrix of bootstrap samples
-  CovMatrix <- cov(cf$t[,ii])
+  CovMatrix <- cf$cf$cov_fn(cf$t[,ii])
```

Comment:
This is one of the reasons why I would prefer to have the effective mass as a mixin to the cf class. That way, the naming would be consistent and you would not need to refer to ```cf$cf``` here. I'm not saying that you should change this now, but it would make for a nicer interface in the long run in my opinion.

Reply:
I see it the other way. The effective mass *has* as a correlation function but it *is* not one. The big problem in this function is that the parameter which is of type `effmass` is called `cf`. If we called if `effmass`, then `effmass$cf$cov_fn` would be okay.